packages/explain-plan-helper/src/index.ts

```diff
@@ -50,10 +51,11 @@ export class ExplainPlan {
 
   get usedIndexes(): IndexInformation[] {
     const ixscan = this.findAllStagesByName('IXSCAN');
+    const expressIxscan = this.findAllStagesByName('EXPRESS_IXSCAN');
```

Comment:
... and this is completely optional, already part of another ticket, but https://jira.mongodb.org/browse/COMPASS-8120 points out that `COUNT_SCAN` should also qualify as a "used-indexes" stage, so that might be easy enough to add here?

Reply:
yeah totally. I'll add it here as well!

Reply:
added in [4a08d34](https://github.com/mongodb-js/compass/pull/6071/commits/4a08d34c9a3f0da7c3a8582ea59525e3fe17ddd9)